pyproject.toml

```
@@ -35,16 +35,16 @@ classifiers = [
 requires-python = ">= 3.11, < 4"
 dependencies = [
   "python-dateutil >= 2.8.2, < 3.0",
-  "typing-extensions == 4.11.0",
+  "typing-extensions == 4.12.0",
```

This should be widened:
```suggestion
  "typing-extensions => 4.11.0, < 5.0.0",
```